docker-compose/database/db-changes/studies/0013_alter_description_type_name.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE study CHANGE public_description description text;
```

Comment:
It seems that text was deprecated:
https://www.dofactory.com/sql/text#:~:text=TEXT%20is%20a%20variable%2Dlength,length%20of%20the%20character%20string.
You can use VARCHAR(Max) instead